src/main/resources/i18n/strings.properties

```diff
@@ -53,6 +53,13 @@ addvaultwizard.new.locationPrompt=…
 addvaultwizard.new.directoryPickerLabel=Custom location
 addvaultwizard.new.directoryPickerButton=Choose…
 addvaultwizard.new.directoryPickerTitle=Select Directory
+addvaultwizard.new.expertSettings.enableExpertSettingsCheckbox=Enable expert settings
+addvaultwizard.new.expertSettings.shorteningThreshold.invalid=Invalid
+addvaultwizard.new.expertSettings.shorteningThreshold.message=Enter a value between 36 and 220 (default 220).
+addvaultwizard.new.expertSettings.shorteningThreshold.tooltip=Open the documentation to learn more.
+addvaultwizard.new.expertSettings.shorteningThreshold.title=Maximum length of encrypted file names
+addvaultwizard.new.expertSettings.shorteningThreshold.valid=Valid
+addvaultwizard.new.expertSettings.summary.introduction=This is a quick summary of your vault information
```

Comment:
Too wordy. I would drop the whole "introduction" thing

Reply:
I second that, especially since this will be likely dropped when the wizard is refactored.

Reply:
introduction is removed